cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -40,16 +37,18 @@ configuration:
 
 imagePullSecrets: {}
 
-registryCaBundleConfig: {}
+registryCaBundleConfig:
+  key: {}
+  name: {}
 
 webhooks:
   enabled: true
 
 rbacManager:
   deploy: true
-  skipAggregatedClusterRoles: true
+  skipAggregatedClusterRoles: false
   replicas: 1
-  managementPolicy: Basic
+  managementPolicy: All
```

Comment:
We diverged from upstream to [improve our security posture](https://github.com/upbound/universal-crossplane/pull/30) with the upbound agent. The upbound agent was running on the UXP clusters and could connect to the Upbound cloud and operate on the cluster hence we wanted to scope the permissions down.

With the [deprecation/removal of the upbound agent](https://github.com/upbound/universal-crossplane/pull/300), there is no reason for this and we can revert this back to the same defaults as upstream.

Introduced with https://github.com/upbound/universal-crossplane/pull/30 
/cc @muvaf @negz

Reply:
I have no issue with syncing this with the upstream value for now. That said, per https://github.com/crossplane/crossplane/issues/4043 I'd actually like to make `Basic` the default in upstream.